packages/core/src/mailers/mailers/adapters/mailgun.ts

```diff
@@ -13,12 +13,15 @@ export type MailgunResponse = {
 export default function createMailgunTransport({
   transportOptions,
 }: MailerOptions) {
-  const domain = env.LATITUDE_DOMAIN
-  const apiKey = env.MAILER_API_KEY
+  const domain = env.LATITUDE_EMAIL_DOMAIN
+  const apiKey = env.MAILGUN_MAILER_API_KEY
 
   if (!domain || !apiKey) return null
```

Comment:
should we raise instead of returning null if the domain env vars aren't defined?